x-pack/plugins/reporting/server/lib/content_stream.ts

```diff
@@ -66,7 +66,9 @@ export class ContentStream extends Duplex {
     return Math.floor(max / 2);
   }
 
-  private buffer = Buffer.from('');
+  private buffers: Buffer[] = [];
+  private currentByteLength = 0;
+
```

Comment:
What do you think if we name it more consistent with the other ones?
```suggestion

    private bytesBuffered = 0;
```